csi_camera.cpp

```diff
@@ -288,19 +287,20 @@ std::vector<unsigned char> CSICamera::get_csi_image() {
 }
 
 std::string CSICamera::create_pipeline() const {
-    std::ostringstream oss;
+    auto device_params = get_device_params(device);
+    std::string input_sensor = (device.value == device_type::jetson) ? (" sensor-id="+video_path) : "";
```

Comment:
Adding conditionality for `input_sensor` which can be used on Jetson devices with two mipi ports.